### PR TITLE
[34901] Use format_time_as_date to get the locale formatted date

### PR DESCRIPTION
**Pull request by oliverguenther**
https://community.openproject.org/work_packages/34901

### CODE REVIEW

**Comment by oliverguenther** [comment]
Hi @HDinger , since you looked at the meeting issue the last time, might I assign this to you?

File: modules/meeting/spec/mailers/meeting_mailer_spec.rb [hunk 1]
```diff
@@ -95,6 +95,38 @@
...
+                          project: project,
+                          start_time: '2021-11-09T23:00:00 +0100'.to_datetime.utc
+      end
+
+      describe 'it renders november 9th for Berlin zone' do
+        let(:mail) { described_class.content_for_review meeting_agenda, 'meeting_agenda', author }
+
+        it 'renders the mail with the correct locale' do
+          expect(mail.text_part.body).to include('11/09/2021 11:00 PM-12:00 AM (GMT+01:00) Europe/Berlin')
+          expect(mail.text_part.body).to include('11/09/2021 11:00 PM-12:00 AM (GMT+01:00) Europe/Berlin')
```
Comment (HDinger) [inline modules/meeting/spec/mailers/meeting_mailer_spec.rb:112]: That should test for the `html_part`

File: modules/meeting/spec/mailers/meeting_mailer_spec.rb [hunk 2]
```diff
@@ -168,6 +200,38 @@
...
+                          project: project,
+                          start_time: '2021-11-09T23:00:00 +0100'.to_datetime.utc
+      end
+
+      describe 'it renders november 9th for Berlin zone' do
+        let(:mail) { described_class.icalendar_notification meeting_agenda, 'meeting_agenda', author }
+
+        it 'renders the mail with the correct locale' do
+          expect(mail.text_part.body).to include('11/09/2021 11:00 PM-12:00 AM (GMT+01:00) Europe/Berlin')
+          expect(mail.text_part.body).to include('11/09/2021 11:00 PM-12:00 AM (GMT+01:00) Europe/Berlin')
```
Comment (HDinger) [inline modules/meeting/spec/mailers/meeting_mailer_spec.rb]: Same as above

**Review by HDinger — COMMENTED** [review]
Looks good 👍 There is just one minor remark regarding the tests

File: modules/meeting/spec/mailers/meeting_mailer_spec.rb (hunk 1 above)
Comment (oliverguenther) [reply modules/meeting/spec/mailers/meeting_mailer_spec.rb:112]: Thanks for catching! :bow:

**Pull request merged**